### PR TITLE
Package size

### PR DESCRIPTION
**Pull request by robogen**
Slight modification to package.json. I don't want to just outright remove electron from it as it removes the local executable files to do testing without outright building the application. This modification just moves the dependency from Dependencies to devDependencies. All packages listed in devDependencies are not packaged into the resultant application, as they are simply there for developing the application. On packaging all the packages listed there are ignored. With this modified package.json the total packaged executable goes from 500+ MB to around and less than 200 MB.

### CODE REVIEW

File: HCCGo/main.js [hunk 1]
```diff
@@ -29,5 +29,6 @@ app.on('ready', function() {
 
     mainWindow.once('ready-to-show', () => {
         mainWindow.show();
+        //mainWindow.webContents.openDevTools();
```
Comment (djw8605) [inline HCCGo/main.js]: Why is this line included?

File: HCCGo/package.json [hunk 2]
```diff
@@ -23,14 +22,17 @@
     "diskusage": ">= 0.1.5",
     "electron-debug": "^1.0.1",
     "fs": ">= 0.0.2",
+    "electron-debug": "^1.0.1",
```
Comment (djw8605) [inline HCCGo/package.json]: This electron-debug package was included 2 lines up already.

**Comment by djw8605** [comment]
Otherwise, this pull request looks good.  The package size is 150MB.

I wonder if we could reduce the size even more... Something to look at later.

File: HCCGo/main.js (hunk 1 above)
Comment (robogen) [inline HCCGo/main.js]: Simple way to turn on/off debugging tools for packaged applications. Release versions has this line commented, Debug versions has this line uncommented.

Comment (djw8605) [inline HCCGo/main.js]: Is there a better way to do this?  Maybe a menu item to open the debug window?

File: HCCGo/package.json (hunk 2 above)
Comment (robogen) [inline HCCGo/package.json]: Accident on my part! Was seeing if keeping that in devDependencies would help reduce total file size, but it caused build issues. Maybe worth looking into again at some point, but for now this is a 'whoops' on my part, I'll commit a fix for this.

File: HCCGo/main.js (hunk 1 above)
Comment (robogen) [inline HCCGo/main.js]: Potentially, it's more I don't want to make those tools something a normal user will have easy access to. If I'm building a debug version application it's something specifically one of us will be using to test an issue found just before version release. Normal users don't really need it, so would rather not make it too easily accessible.

Comment (djw8605) [inline HCCGo/main.js]: Eh, having it in a "help" menu or something would be ok with me.

Anyways, I'll open an issue so we can discuss this further.  No need to hold up this pull request.
